build_projects/dotnet-cli-build/PackageDependencies.cs

```diff
@@ -1,96 +0,0 @@
-﻿using System;
```

Comment:
Yes it was in both dotnet-cli-build and shared-build-targets-utils. And was causing build warnings.

I kept the shared-build-targets-utils one.